LayoutKit/LayoutArrangement.swift

```diff
@@ -151,8 +151,9 @@ extension LayoutArrangement: CustomDebugStringConvertible {
         let t = String(repeatElement(" ", count: indent * 2))
         let sublayouts = self.sublayouts.isEmpty
```

Comment:
Do you need to check for isEmpty? I think the map/joined expression would produce the correct result in both cases and we don't need special handling for the empty case.